CMakeLists.txt

```diff
@@ -179,6 +178,16 @@ core_optional_dep(${optional_deps})
 core_require_dyload_dep(${required_dyload})
 core_optional_dyload_dep(${dyload_optional})
 
+if(ENABLE_MYSQLCLIENT AND ENABLE_MARIADBCLIENT)
+    MESSAGE(FATAL_ERROR "You can not use MySql and MariaDB at the same time. Disable one by adding -DENABLE_MYSQLCLIENT=OFF or -DENABLE_MARIADBCLIENT=OFF")
+endif()
```

Comment:
This logic has to be adjusted.
If you don't disable any of them at the second cmake run this error message shows up, which shouldn't happen.